.vscode/settings.json

```diff
@@ -21,5 +21,46 @@
   "solidity.packageDefaultDependenciesDirectory": "lib",
   "solidity.compileUsingRemoteVersion": "v0.8.17+commit.8df45f5f",
   "solidity.formatter": "prettier",
-  "solidity.defaultCompiler": "remote"
-}
+  "solidity.defaultCompiler": "remote",
+  "tailwindCSS.classAttributes": [
+    ".*class.*",
+    ".*Class.*",
+    ".*Properties.*",
```

Comment:
Preference here is to lay out the class names that we'd use rather than have all the regexes below. That makes all this more decipherable and if someone wants a new class name they can just add it here (otherwise `className`, `buttonClassName`, and so on should cover most things).